app/views/idv/shared/_back.html.erb

```diff
@@ -5,13 +5,15 @@ path can be passed in case the HTTP header is not specified or is invalid. If no
 yield a useable URL, nothing will be rendered.
 
 locals:
+* flow: (Optional, but required with step/action) Name of the flow for the action/step
 * action: (Optional) Flow action to call to return to the previous step.
 * step: (Optional) Name of step to which user should be returned.
 * fallback_path: (Optional) Path to redirect absent action, step, and HTTP referer.
 %>
 <% text = '‹ ' + t('forms.buttons.back')
+   flow = local_assigns[:flow]
    step = local_assigns[:action] || local_assigns[:step]
-   path = step ? idv_doc_auth_step_path(step: step) : go_back_path
+   path = (flow && step) ? method(:"idv_#{flow}_step_path").call(step: step) : go_back_path
```

Comment:
In other places we've used the `@step_url` instance variable assigned by FLowStateMachine. Can we do that here as well, to make sure we're consistently using the flow's `step_url` configuration? 

https://github.com/18F/identity-idp/blob/1986d428e1f644f278a94d70f7061e728ecb3760/app/views/idv/shared/_document_capture.html.erb#L26

https://github.com/18F/identity-idp/blob/1986d428e1f644f278a94d70f7061e728ecb3760/app/services/flow/flow_state_machine.rb#L76

Reply:
The remote proofing address page didn't like using `@step_url`, so I made it support `local_assigns[:step_url]` as well.